### PR TITLE
Icon has been added to FormSelect component

### PR DESCRIPTION
**Pull request by Afshin-Karamifar**

### CODE REVIEW

File: example/src/components/FormSelectWithIconDemo.tsx [hunk 1]
```diff
@@ -61,7 +63,7 @@ export const FormSelectWithIconDemo = () => {
                 value: 'Frog',
                 ariaLabel: 'Frog',
                 id: 'id3',
-                icon: '/capgemini.png',
+                //icon: '/capgemini.png',
```
Comment (Ibabalola) [inline example/src/components/FormSelectWithIconDemo.tsx:66]: if commented out, should we remove or do you have questions regarding these props

File: stories/FormSelect/UnStyled.stories.js [hunk 2]
```diff
@@ -39,18 +39,18 @@ export const WithIcon = {
...
+      selectWidth: '185px',
+      listItemsCountToShow: 7,
+      itemTextColor: '#000000',
+      border: '1px solid #747d8c',
+      itemBackgroundColor: '#ffffff',
+      itemDisabledTextColor: '#747d8c',
+      groupTilteBackgroundColor: '#0000000D',
+      itemHoverBackgroundColor: '#0000000D',
+      fontFamily: '"GDS Transport", arial, sans-serif',
+      // selectStyle: {{}},
```
Comment (Ibabalola) [inline stories/FormSelect/UnStyled.stories.js:52]: if commented out, should we remove or do you have questions regarding these props

File: stories/FormSelect/UnStyled.stories.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ export const WithIcon = {
             value: 'Invent',
             ariaLabel: 'Invent',
             id: 'id2',
-            icon: '/capgemini.png',
+            // icon: '/capgemini.png',
```
Comment (Ibabalola) [inline stories/FormSelect/UnStyled.stories.js:77]: if commented out, should we remove or do you have questions regarding these props

File: stories/liveEdit/FormSelectLive.tsx [hunk 4]
```diff
@@ -29,25 +33,26 @@ function FormSelectDemo() {
...
+          selectWidth: '185px',
+          listItemsCountToShow: 7,
+          itemTextColor: '#000000',
+          border: '1px solid #747d8c',
+          itemBackgroundColor: '#ffffff',
+          itemDisabledTextColor: '#747d8c',
+          groupTilteBackgroundColor: '#0000000D',
+          itemHoverBackgroundColor: '#0000000D',
+          fontFamily: '"GDS Transport", arial, sans-serif',
+          // selectStyle: {{}},
```
Comment (Ibabalola) [inline stories/liveEdit/FormSelectLive.tsx:46]: if commented out, should we remove or do you have questions regarding these props

File: stories/liveEdit/FormSelectLive.tsx [hunk 5]
```diff
@@ -29,25 +33,26 @@ function FormSelectDemo() {
...
+          // listStyle: {{}},
           iconStyle:{
             width: '18px',
             height: '18px',
             borderRadius: '0px'
           }
         }
       }
+      // options={['Capgemini', 'Cybersecurity', 'Intelligent Industry', 'Sustainability', 'Cloud']}
```
Comment (Ibabalola) [inline stories/liveEdit/FormSelectLive.tsx:55]: if commented out, should we remove or do you have questions regarding these props

File: stories/liveEdit/FormSelectLive.tsx [hunk 6]
```diff
@@ -92,7 +98,7 @@ function FormSelectDemo() {
               value: 'Invent', 
               ariaLabel: 'Invent', 
               id: 'id2',
-              icon: '/capgemini.png',
+              //icon: '/capgemini.png',
```
Comment (Ibabalola) [inline stories/liveEdit/FormSelectLive.tsx:101]: if commented out, should we remove or do you have questions regarding these props

Comment (Afshin-Karamifar) [reply stories/liveEdit/FormSelectLive.tsx]: Thanks for your comment, removed.

File: stories/liveEdit/FormSelectLive.tsx (hunk 5 above)
Comment (Afshin-Karamifar) [reply stories/liveEdit/FormSelectLive.tsx:55]: Thanks for your comment, This is just an example of using options, and if anyone want to test it in live mode they don't need to write code, by removing back slashes they can see the result in live.  same as options with icon and optiongroup.

File: stories/liveEdit/FormSelectLive.tsx (hunk 4 above)
Comment (Afshin-Karamifar) [reply stories/liveEdit/FormSelectLive.tsx]: Thanks for your comment, back slashes removed.

File: stories/FormSelect/UnStyled.stories.js (hunk 3 above)
Comment (Afshin-Karamifar) [reply stories/FormSelect/UnStyled.stories.js]: Thanks for your comment, removed.

File: stories/FormSelect/UnStyled.stories.js (hunk 2 above)
Comment (Afshin-Karamifar) [reply stories/FormSelect/UnStyled.stories.js]: Thanks for your comment, back slashes removed.

File: example/src/components/FormSelectWithIconDemo.tsx (hunk 1 above)
Comment (Afshin-Karamifar) [reply example/src/components/FormSelectWithIconDemo.tsx]: Thanks for your comment, removed.

File: example/src/components/FormSelectWithIconDemo.tsx [hunk 7]
```diff
@@ -17,20 +19,20 @@ export const FormSelectWithIconDemo = () => {
...
+          selectWidth: '185px',
+          listItemsCountToShow: 7,
+          itemTextColor: '#000000',
+          border: '1px solid #747d8c',
+          itemBackgroundColor: '#ffffff',
+          itemDisabledTextColor: '#747d8c',
+          groupTilteBackgroundColor: '#0000000D',
+          itemHoverBackgroundColor: '#0000000D',
+          fontFamily: '"GDS Transport", arial, sans-serif',
+          // selectStyle: {{}},
```
[the remark this answers is not in the document]
Comment (Afshin-Karamifar) [reply example/src/components/FormSelectWithIconDemo.tsx]: Thanks for your comment, back slashes removed.

**Pull request closed without merging**